src/main/kotlin/blackjack/domain/game/BlackJackGame.kt

```diff
@@ -0,0 +1,34 @@
+package blackjack.domain.game
+
+import blackjack.domain.card.Deck
+import blackjack.domain.player.Players
+import blackjack.views.OutputView
+
+class BlackJackGame(private val players: Players, private val deck: Deck) {
+
+    fun play(
+        isPlayerTurnOff: Turn,
+        initCallback: (Players) -> Unit,
+        playingCallback: (Players) -> Unit,
+    ): Players {
+        val afterInitPhased = players.startInitPhase(deck)
+        initCallback(afterInitPhased)
+        val playingPhasedPlayers = playingPhase(isPlayerTurnOff, deck, afterInitPhased)
+        playingCallback(playingPhasedPlayers)
+        return playingPhasedPlayers
+    }
```

Comment:
```suggestion
class BlackJackGame(private val players: Players, private val deck: Deck) {

    fun play(
        isPlayerTurnOff: Turn,
        initCallback: (Players) -> Unit,
        playingCallback: (Players) -> Unit,
    ) {
        val afterInitPhased = players.startInitPhase(deck)
        initCallback(afterInitPhased)
        val playingPhasedPlayers = playingPhase(isPlayerTurnOff, deck, afterInitPhased)
        playingCallback(playingPhasedPlayers)
        this. players = playingPhasedPlayers
    }
```
players를 blackjackgame에서 관리하는게 좋지 않을까요.  그렇지 않다면 생성자로 전달 받기보단 play 파라미터로 전달 받아야 될거같아요.
player를 관리한다면 BlackJackGame이 스코어를 계산하는 주체가 될수도 있을거 같아요.